Setup/UpgradeData.php

```diff
@@ -571,6 +571,41 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
             }
         }
 
+        if (version_compare($context->getVersion(), '4.2.0', '<=')) {
+        // Set a Age check 18+ for this product.
```

Comment:
indenting klopt niet helemaal in dit blok

Reply:
done